values-schema.yaml

```diff
@@ -667,12 +667,51 @@ properties:
             type: array
             items:
               type: string
-      gatekeeper-operator:
+      gatekeeper:
         type: object
         additionalProperties: false
         properties:
           enabled:
             type: boolean
+          constraints:
```

Comment:
this won't be used as such...please next time don't waste so much precious time describing solutions that have not been approved. You really have to start scratching yourself behind the ears: is this what we will end up using or delivering? What do the reviewers say about it?

Reply:
Well, this started with a blindfolded design from the start, so what we have right now can be adjusted in every way to get aligned with the specs.